backend/build.gradle

```diff
@@ -2,12 +2,18 @@ plugins {
 	id 'org.springframework.boot' version '2.3.5.RELEASE'
 	id 'io.spring.dependency-management' version '1.0.10.RELEASE'
 	id 'java'
+	id 'checkstyle'
+	id 'jacoco'
+	id 'org.sonarqube'  version '2.7.1'
```

Comment:
This is a little aspirational, but we _could_ actually set this up on the free site if we want to.

Reply:
I would be in favor of that, I've used it on a bunch of other projects, happy to set that up.

Reply:
I'd also be in favor but don't think it's in my top 10 tasks for this week, so I will add an issue for it and we can do it when we can do it.